### PR TITLE
LPD-34742 Fixed Project selection validations, and added Marketplace support email

### PR DESCRIPTION
**Pull request by welitonpe**

### CODE REVIEW

**Comment by welitonpe** [comment]
ci:test:sf

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/CloudProvisioning/pages/Installation.tsx [hunk 1]
```diff
@@ -20,15 +20,15 @@ enum Statuses {
...
 const statuses = {
 	[Statuses.FAILED]: {
 		bodyMessage: (
 			<span>
 				Sorry, was not possible to install your app, you can try again.
 				If the problem persist, contact a{' '}
-				<a href="mailto:marketplace@liferay.com">
-					marketplace@liferay.com
-				</a>
+				<a href={`mailto:${MARKETPLACE_ADMIN_EMAIL}`}>marketplace@liferay.com</a>
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/CustomerDashboard/pages/Apps/App/CloudProvisioning/pages/Installation.tsx:31]:
```suggestion
				<a href={`mailto:${MARKETPLACE_ADMIN_EMAIL}`}>{MARKETPLACE_ADMIN_EMAIL}</a>
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
Just this

**Comment by welitonpe** [comment]
ci:test:sf

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Pull request closed without merging**